src/ARCtrl/ARC.fs

```diff
@@ -570,6 +572,39 @@ type ARC(?isa : ArcInvestigation, ?cwl : CWL.CWL, ?fs : FileSystem.FileSystem) =
             ARCtrl.Json.ARC.ROCrate.encoder (Option.get obj.ISA)
             |> ARCtrl.Json.Encode.toJsonString (ARCtrl.Json.Encode.defaultSpaces spaces)
 
+    /// <summary>
+    /// Returns the write contract for the input ValidationPackagesConfig object.
+    ///
+    /// This will mutate the ARC file system to include the `.arc/validation_packages.yml` file if it is not already included.
+    /// </summary>
+    /// <param name="vpc">the config to write</param>
+    member this.GetValidationPackagesConfigWriteContract(vpc: ValidationPackagesConfig) =
+        let paths = this.FileSystem.Tree.ToFilePaths()
+        if not (Array.contains ValidationPackagesConfigHelper.ConfigFilePath paths) then
+            Array.append [|ValidationPackagesConfigHelper.ConfigFilePath|] paths
+            |> this.SetFilePaths
+        ValidationPackagesConfig.toCreateContract vpc
+
+    /// <summary>
+    /// Returns the delete contract for `.arc/validation_packages.yml`
+    /// </summary>
+    /// <param name="vpc"></param>
+    member this.GetValidationPackagesConfigDeleteContract(vpc) =
```

Comment:
This does not check if it must be removed from filesystem?

Reply:
Was not sure on the logic of this. So we remove the file system entry although the contract has not been fulfilled?

Reply:
Yes, as we return the contract we must assume it will be fullfilled. We also do this for RemoveAssay, etc.

Reply:
okay, done 👍